addok_psql/processors.py

```diff
@@ -6,11 +6,17 @@
 import psycopg2
 import psycopg2.extras
 
-from addok import config
+from addok.config import config
 from addok.helpers import yielder
 
 
-connection = psycopg2.connect(**config.PSQL)
+connection = psycopg2.connect(
```

Comment:
What's the rationale behind this change? Doing so will prevent using any specific [connect](http://initd.org/psycopg/docs/module.html?highlight=connect#psycopg2.connect) argument afaik

Reply:
I wanted to keep parameter names from config unchanged, and the kwargs in connect have not the same name as described there: http://initd.org/psycopg/docs/module.html

Reply:
(actually before this change I had an error on 'dbuser' kwarg which does not exist in connect)

Reply:
Anyway, we can just keep **config.PSQL and change config if you prefer.

Reply:
Hm. I may be wrong on this one, it seems my parameters are not correctly handled.

Reply:
Ok, will delay it for another PR, after better testing.